### ISSUE TITLE
A call for source code

### ISSUE THREAD

**Issue by dankennard** [Nibre/MotherVR#53]
Whilst it is understandable that the project author would prefer for the code to be cleaned up before going open source, GitHub is not a file-hosting service - the project source code should be made available accordingly or the project should be hosted elsewhere.

**Comment by steren** [comment]
GitHub does not requires the author to release the source code. It is fine to only use GitHub to distribute releases and collect issues.

However, the author should really consider making the source code available in this repo. This would speed us the mod's creation as other people could send Pull Requests to fix the many known bugs

**Comment by nlooije** [comment]
@Nibre you should really consider releasing the source code sooner rather than later. That way others can help.

**Comment by d0x360** [comment]
@Nibre having others be able to help would probably be of huge benefit.  You started this so the glory is yours, now imagine how much bigger it could be with help.  There could be people working on it 24/7.

**Comment by d0x360** [comment]
It wouldn't be a bad idea to make it public.  Having more eyes looking at
it would help solve problems way quicker plus we all know who started this
and he gets the props no matter what

**Comment by NeonUniverse** [comment]
True, but then again it is his choice. Still I do support the idea but not until the future. Maybe at V1 release or something...

**Comment by alex9099** [comment]
Still no answer from @Nibre ?

**Comment by burzum** [comment]
@Nibre still alive? Please make this open source so other people can continue to work on it if you lost the interest or have no more time.

**Comment by JoshWobbles** [comment]
There was a release on June 11, so obviously still being worked on.

**Comment by alex9099** [comment]
That was almost 4 months ago ;) i don't mean that there should be updates every day, but the project seems quite stuck, assuming that, as you said, last release was at june 11.

It would be nice to have a build for linux though :)

**Comment by Wokstar** [comment]
The release notes for Beta 0.6.0 state that the project is close to open source release. I think it makes sense to wait until Motion Control Support is implemented before releasing the source as that's a good milestone. 

Lets not forget that this is a labour of love, and the last release was huge. Read the release notes for things currently being worked on.

**Comment by d0x360** [comment]
Neon I understand where you're coming from and I 100% agree it's his choice BUT a small community has (excuse the pun) developed around this mod and as a community I don't see an issue with people asking.

It's not pressuring the guy, he can just not read any topic that is asking to make it open source early.  The only reason we want it opened up is so we can help.
